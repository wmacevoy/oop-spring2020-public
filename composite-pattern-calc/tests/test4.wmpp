((((b*q)+v)+(((((((l+z)+6)*((i+m)*5))*0)+((((c+3)+61)+((815+151)*(t*1)))+((1*(d*l))*((4*p)+(0+9)))))+(((20+((y*7)*(587*r)))*(((305+1)+(5+5))*h))*d))*v))*((6+((((((e*7)+(491*u))*((f*9)*(53+833)))+(((u+w)+(u*r))*56))*((k*((w+p)*(p+6)))*((1+(3+c))*8)))+(((((n*6)*(m+6))+((r+1)+(e*g)))+(((0*3)*(6+3))+((833+b)+4)))+((((1*e)+(5*8))+(5*(o+782)))*(0+s)))))+(((((6+q)+(((h+l)+(2*d))+(e*u)))*t)+(2*(l*((t*(y+0))+((h*o)+(9+h))))))+(((((0*(9+k))+((h+409)+(a+j)))+((p*(v*c))+((7*6)*(670*3))))*((5+7)+(((n+335)+(7+z))+((u+2)+(x*804)))))+((((g+9)*((5*v)+(p*r)))*(((h*53)*(4+589))*((4+7)+(d*5))))*e)))))
(((((((((9+d)*z)+(t*(a*3)))*(((3+l)+(d*f))+((5*l)*(8+816))))+((41+((n*7)*(x*n)))*(((v+s)+o)*(b*(c*f)))))+(((((r*6)*(9+m))+((8+e)+(p+282)))*1)*m))*((((((p*7)+(d*s))*((d*5)+(f*q)))+(((z+p)+(c*u))*((3+k)*(2+175))))+((7*g)+((5*(t*0))*((7*p)+y))))+(((q+((3+b)+(g+z)))*(((9*m)+(3*g))*((0+9)+1)))+(((2*(8*s))*((6*k)+(y*3)))*o))))*n)*((((((((e+9)+(8+v))+((7+t)+4))*(((w*c)*(2*g))*r))*(7+((j*(247+7))+((0*2)*(h*3)))))+(((i+((5*f)*(o+9)))+(((414+m)+(v+757))*((z+370)+(n+6))))+(4+(((928*h)+985)*((9+7)*(a+2))))))+7)+((s+(((((i+7)+(g*n))*((a*426)+(j+r)))+o)+((((r+773)+(n+148))*((4+e)+j))*(((a+379)+(c*611))*((x*s)*e)))))*((((((b*5)+(5*197))*((0*u)*(0*3)))+(((3*376)+(m+2))+((x+3)*(u+h))))*(6+(o*((0*e)+(v+8)))))*(((j+g)*(((x+j)+(6+0))+(7*(j+d))))+((((h*2)+(5+l))+((3*b)+(z+z)))+(((h*d)+a)*((964+0)+(2+9)))))))))
(((((2+(((6+643)*7)+(7+((504+5)+(b*78)))))+((((s+(7*1))+9)+(((h+9)+16)+2))+((((z*f)*r)*(237+(3*t)))+(3*((i*y)*(t*6))))))*((((413*(1*(596+b)))*(((z+6)*x)*((f*q)*(g*4))))+(x+((63+(f*f))*9)))+(((((138+7)+(4+b))*((7*q)*d))+(((7*m)*6)*((6+r)+(d+h))))*a)))*((p*(((((123*v)*(7+k))+((2+0)+7))*(((3*530)+(b+0))*((j+f)+(9*a))))*1))*l))*(0*(((((((j*b)*(t+r))*((b+o)+(e+h)))+(9+a))*(((5*(4+e))*966)+(f+((r+7)*(u*3)))))*(709+(((868*(p+w))+w)*(((5+721)+(l*c))*578))))+((((((p+6)*(f*1))*((b+3)*(282+s)))*(((a*z)+(c*k))+r))*((((w+2)+(6*g))+(k+(u+f)))+((w+(b*b))+((325*x)+(3+0)))))*(((((x+9)+(f+5))*((413+8)*(8*o)))*((o+(5+n))*((11*957)*(4*o))))*u)))))
(f+l)
((((((726+s)+((((h+n)*0)+((n*656)*(2+x)))*9))+(((((x*3)+(g*g))*c)*((y+(1+616))*(1*u)))*(((x+(1+481))+((7*1)*(a+r)))+((h+(h*14))*((w*2)+(s*254))))))*((d*r)+(((((13*y)+6)*5)*(q*(4*(1*7))))*((4*((27*y)+(a*m)))*(((m+h)*(r*z))+((i+t)*(8*f)))))))+(((((((7+7)*d)+((0+b)+(v+327)))+2)+(436+681))+(9+(((2+(793*281))*((887*n)+v))*(((d*234)*8)*((y*9)*(d+4))))))+(((((f*(l*d))*((7+g)*(k*x)))*(m*((8+b)*(0+0))))+7)+(5+(((2+(157+7))*((0+e)*233))+(((1+n)*(l*l))+((1+1)+(w*q))))))))+(((((w+(9*((6+n)+(3*3))))+1)*(((((h+0)+m)+(2*(474+4)))*(((796+s)+(4*3))*(582*(3*r))))+(250+(g*((6*w)+(g*8))))))+((((z+((1+1)*7))*((i+(x*o))+(963*r)))*g)*(((((8+p)+(307+t))*((5+p)*(3+7)))*895)*v)))*((j*(((((0+n)*(k*6))*5)+(((z*p)+(947+w))*((y*5)*(f+3))))+h))*(((x*(((882*8)*(4+u))+((w+y)*(2+3))))*(((w*(c*3))*o)*(((g+2)*(g+573))+((t*m)+(m*403)))))+(((((t*776)+s)+((l+g)*(c*e)))+((911+(f*u))+((9+1)*(8+1))))*((((654*y)*(4+l))+((3*k)*(1*0)))+(z+g)))))))
l
3
(d+((((((((u+8)*(6*w))+(4+(7*9)))+((3*v)*((2*2)*(4+b))))+((((q*l)*(l*628))*((i+4)*p))*((v+(q*817))+((v+n)*(461+738)))))+7)*7)*3))
((((((((z*(314+z))+((986+9)*(o+362)))*(((2*6)*(1+1))+((6+473)+(i+7))))*464)*(((r*((p+u)+(c*95)))+j)+928))+((((((630+9)*7)+((d*530)*(0*x)))*b)*j)+(((s*((w*m)*(d*j)))*(((t*5)+131)+((k+b)*843)))*((((5*331)*132)+((5*j)*(7*2)))*((k*7)+((6+319)*(t+w)))))))+((((((569+z)+((1+6)*9))*7)+((((291+3)*2)+n)+8))*(((((p*t)*(8+9))*((578*4)*y))*(((9+2)+(b*g))*((p*c)+q)))+(9+3)))+((5+((((x+0)*(1+w))+((z*2)+(6+5)))+(((r*i)*r)*((t+x)*(4+n)))))+((l*(4+o))+((((3+2)*h)*((839+382)+(4+687)))+(((6*9)+(6+f))*401))))))*(r+(((u+(4*(((z*385)+(m+4))*((772*758)+(u+8)))))+(((((j+3)+(x*b))*((z*3)+z))+(((d*4)*s)+(5*(5*q))))+((4*8)*(((6*n)*(1+h))*616))))+z)))
(1*((((780*((((1*9)*(0*8))+o)+(642+(y*(b*5)))))*(((((8+2)+9)*m)*3)*((509*968)+(4*((7*u)*(0*2))))))*((((l+((376*93)+(t*5)))+(((d+430)+n)*((x*i)+s)))+(((3+(7*1))*(l+m))*(((g*7)*(4+3))*((9*1)*(r*1)))))*(((((n*m)*(o+d))*((8+8)*(u+8)))+(((i+5)+l)+a))*(((5+(0*y))*s)+(f+((b*h)*(404+4)))))))+(7*((7+j)*(((((4*w)+j)+(2+x))*((593+(x+4))*((c+9)+(s+c))))+((((4+4)*(8+j))+x)*(((g+v)*(y*3))+((b+981)+(s+q)))))))))
