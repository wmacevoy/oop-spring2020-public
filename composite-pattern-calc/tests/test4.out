1.9287922E7
2517251.8
0.0
0.019063182
3.9108307E13
0.009259259
3.0
1571.064
1.3267516E21
2.11761913E15
